packages/xchain-binance/src/client.ts

```diff
@@ -219,7 +219,7 @@ class Client implements BinanceClient, XChainClient {
         })
         .filter((balance) => !asset || balance.asset === asset)
     } catch (error) {
-      return Promise.reject(error)
+      return [] as Balances
```

Comment:
It might not a good idea to return an empty list of balances for ALL kind of errors. Is there a way to check the error to decide whether to return `[] as Balances` (for null balances accounts only) or `Promise.reject(error)` for all other errors?